app.js

```diff
@@ -265,27 +298,38 @@ function on_yt_logo_clicked() {
 
 
 function run_extension_functions() {
-    console.log("[Better Mobile Youtube] Extension Functions now runing");
+    console.log("[Better Mobile Youtube] Extension Functions now running");
 
     // setTimeout(on_video_page, 1500);
 
     // setTimeout(autoplay_video, 500);
 
-    // setTimeout(on_player_settings_clicked, 3000);
+    /* setTimeout(on_player_settings_clicked, 3000); 
+    Function is called in on_video_played() */
 
     // setTimeout(load_custom_playback_speed, 2000);
 
     // setTimeout(change_youtube_logo, 1000);
 
-    setTimeout(on_yt_logo_clicked, 1000);
+    setTimeout(on_yt_logo_clicked, 200);
 
     // setTimeout(on_video_played, 1000);
 
     // setTimeout(open_link_in_new, 1500);
 
-    setTimeout(loop_to_check_player_visibility, 1000);
+    setTimeout(loop_to_check_player_visibility, 200);
 
 
+    // Check when the url changes and insure that the actual webpage has changed, if yes run loop_to_check_player_visibility again
```

Comment:
could this be made into a function and then, the function be called inside run_extension_functions()